includes/core/server/request/class-get-reporting-payment-activity.md

```diff
@@ -13,26 +13,31 @@ The `WCPay\Core\Server\Request\Get_Reporting_Payment_Activity` class is used to
 | `date_start`| `set_date_start( string $date_start )`    |    No     |    Yes   |       -       |
 | `date_end`  | `set_date_end( string $date_end )`        |    No     |    Yes   |       -       |
 | `timezone`  | `set_timezone( string $timezone )`        |    No     |    Yes   |       -       |
+| `currency`  | `set_currency( string $currency )`        |    No     |    Yes   |       -       |
 
 The `date_start` and `date_end` parameters should be in the 'YYYY-MM-DDT00:00:00' format.
 The `timezone` parameter can be passed as an offset or as a [timezone name](https://www.php.net/manual/en/timezones.php).
+`currency` should be a currency code for a store deposit currency.
 
 ## Filter
 
 When using this request, provide the following filter and arguments:
 
 - Name: `wcpay_get_payment_activity`
+- ? what
 
 ## Example:
 
 ```php
+// Does this example explain / add any value?
```

Comment:
It helps when someone needs a quick example. Added due to the precedent as all other classes are documented in this way.

Reply:
Do you think the precedent is worthwhile? What documentation do you think is most useful and worth maintaining? Where should those docs live?

Reply:
Yes, it has been useful to me several times in the past when I was trying to make a request using existing request classes. 

I think there could be improvements to the location and content of the docs. Will plan to create an RFC/ issue to discuss this.